### DISCUSSION TITLE
Why is the contact pressure between pellets and cladding almost zero when the pellet and cladding are clearly deformed?

### DISCUSSION THREAD

**Discussion by Mrxiongjy** [idaholab/moose#27972] [Q&A General] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

I have not set the boundary conditions for the coolant pressure, and I have not set the initial charge pressure of the air chamber, is this the reason?
![b277b81fcb81eed11e69e91405046de3](https://github.com/idaholab/moose/assets/113361478/8e891419-8195-40da-898a-54e07703fd2f)

**Comment by GiudGiud** [comment]
> I have not set the boundary conditions for the coolant pressure, and I have not set the initial charge pressure of the air chamber, is this the reason?

can you try?
This seems like a possible reason to me

**Reply by Mrxiongjy** [reply, marked as answer]
‘’Note that the contact_pressure variable is added but not used with the mortar formulation. The physical meaning of the Lagrange multiplier is the normal contact pressure.‘’ 
I see why. In Mortar contact, the contact pressure is expressed in terms of Lagrange multipliers
